packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -487,11 +543,40 @@ export function makeVirtualObjectManager(
       if (initializing) {
         cache.remember(innerSelf);
       }
-      const representative = Far(tag, actualize(wrappedData));
+      const self = actualize(wrappedData);
+      let storedRepresentative;
+      let exposedRepresentative;
+      const facetiousness = assessFacetiousness(self);
+      switch (facetiousness) {
+        case 'one': {
+          storedRepresentative = Far(tag, self);
+          vrm.checkOrAcquireFacetNames(kindID, null);
+          exposedRepresentative = storedRepresentative;
+          break;
+        }
+        case 'many': {
+          exposedRepresentative = {};
```

Comment:
I tried renaming these in terms of the roles the values were playing rather than trying to pin down the noun, calling them `toHold` (meaning: what the vom will hold in memory) and `toExpose` (meaning: what the vom will reveal to user code).  It reads nicely.